### ISSUE TITLE
npm-install fails at line 250: rc-service openresty start

### ISSUE THREAD

**Issue by myxal** [ej52/proxmox-scripts#66]
**Script**
nginx-proxy-manager
* setup.sh and also npm-install.sh

**Describe the bug**
Setup fails at the step where openresty is launched, reporting that config test failed:
> [error] Starting services
/tmp/npm_install.sh: line 250: rc-service openresty start
14
configuration file /usr/local/openresty/nginx/conf/nginx.conf test failed
openresty failed to start

**System info:**
 - Proxmox Version: 7.1.10
 - Alpine Version: 3.15-default_20211202

**Additional context**
* Set up the lxc container myself, and added /data folder (migrated from an NPM setup on docker, which I wasn't happy with) before installation
* container has 2 GB of disk space and 512 MiB RAM allocated - I hope that's enough

[nginx.conf.txt](https://github.com/ej52/proxmox-scripts/files/8053783/nginx.conf.txt)

**Comment by dhzl84** [comment]
Did you manage to perform the update in the end?

**Comment by myxal** [comment]
Yes, but only because I performed a clean installation. My configuration in the container was not huge and didn't warrant me manually editing the files to make the "update" path to finish successfully.

I'm not sure how desirable it is for you/this project, but I think being able to import config from the docker container, which I consider "upstream" version of NPM, would be helpful.

**Comment by dhzl84** [comment]
Couldn't manage to do a clean install with Alpine, always failed at openresty.
Installation into Debian worked for me.
